src/ui/components/NetworkMonitor/HeaderGroups.tsx

```diff
@@ -35,7 +35,7 @@ export function HeaderGroups({
       )}
       {headerGroups.map((headerGroup: HeaderGroup<RequestSummary>) => (
         <div
-          className="flex items-center divide-x font-normal"
+          className="flex items-center divide-x divide-themeTextField font-normal border-"
```

Comment:
what are these new classes doing?

Reply:
I learned a new thing! [divide-x](https://tailwindcss.com/docs/divide-width) controls the border width between elements and we were using them for the vertical grabby columns in our net monitor.

 So this change is changing the colour of them from very bright gray/white to being contextual based on the theme.

Reply:
And then the border- was just a typo, so I just removed it.